samples/meetings-sidepanel/csharp/Side Panel/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "MicrosoftAppId": "<<Microsoft-App-ID>>",
+  "MicrosoftAppPassword": "<<Microsoft-App-Secret>>",
+  "BaseUrl": "<<Bot-EndPoint-URL>>",
+  "ServiceUrl": "<<Service-URL>>"
```

Comment:
This is not required to be part of configuration. When App is installed to the meeting, in conversation update event you can fetch this info.